orcid-persistence/src/main/java/org/orcid/persistence/dao/impl/WorkDaoImpl.java

```diff
@@ -397,7 +399,24 @@ public List<Object[]> getWorksByOrcid(String orcid) {
                 .addScalar("assertion_origin_client_source_id", StringType.INSTANCE)
                 .addScalar("date_created", TimestampType.INSTANCE)
                 .addScalar("last_modified", TimestampType.INSTANCE)
-                .addScalar("top_contributors", StringType.INSTANCE);
+                .addScalar("contributors", StringType.INSTANCE)
```

Comment:
Please add a //TODO so in the future, after the CLI finishes, we fetch only the top_contributors_json